go/process/health_dao.go

```diff
@@ -93,17 +93,22 @@ func RegisterNode(extraInfo string, command string, firstTime bool) (healthy boo
 		}
 	}
 	{
+		db_backend := ""
+		if config.Config.IsSQLite() {
+			db_backend = config.Config.SQLite3DataFile
+		} else {
+			db_backend = config.Config.MySQLOrchestratorHost + ":" +
+				fmt.Sprintf("%d", config.Config.MySQLOrchestratorPort)
```

Comment:
nitpick, but the `golang` way, since you've already chosen `fmt`, is:
```
db_backend = fmt.Sprintf("%s:%d", config.Config.MySQLOrchestratorHost, config.Config.MySQLOrchestratorPort)
```

while at it, please change `db_backend` to `dbBackend`. `golang`'s naming conventions use camelBack